drivers/hmis_external_apis/app/models/hmis_external_apis.rb

```diff
@@ -0,0 +1,6 @@
+module HmisExternalApis
+  def self.table_name_prefix
+    # FIXME: what's the convention in this app
+    # 'hmis_external_api_'
```

Comment:
@gigxz What's your take on table prefixes? I know there are a ton of tables in this app, so maybe we should have one.

Reply:
Yeah, I've mostly been leaving out the table name prefixes.  I think it's fine to just go with `external_` for these.  If you want to make that an explicit prefix rather than encoded in each model, that's fine, but not necessary.